src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -398,24 +430,30 @@ public Type GetReturnType(string invocationId)
 
             public Type[] GetParameterTypes(string methodName)
             {
-                if (!_connection._handlers.TryGetValue(methodName, out InvocationHandler handler))
+                if (!_connection._handlers.TryGetValue(methodName, out List<InvocationHandler> handlers))
                 {
                     _connection._logger.MissingHandler(methodName);
                     return Type.EmptyTypes;
                 }
-                return handler.ParameterTypes;
+                if (handlers.Count > 0)
```

Comment:
Write a comment about the fact that we're using the first handler to determine the parameter types.